build.gradle

```diff
@@ -14,33 +15,29 @@ repositories {
 
 }
 
-
 def jacksonVersion = "2.10.3"
 def jupiterVersion = "5.6.0"
 def awsSdkVersion = "1.11.791"
 
 dependencies {
-    implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
-    implementation group: 'com.amazonaws', name: 'aws-lambda-java-events', version: '2.2.7'
-    implementation group: 'com.amazonaws', name: 'aws-java-sdk-lambda', version: awsSdkVersion
-    implementation group: 'com.amazonaws', name: 'aws-java-sdk-dynamodb', version: awsSdkVersion
-
-    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jdk8', version: jacksonVersion
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: jacksonVersion
-    implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-parameter-names', version: jacksonVersion
-
 
-    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '0.3.5'
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk-dynamodb', version: awsSdkVersion
+    implementation(group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '0.3.5')
+    implementation(group: 'org.slf4j', name: 'slf4j-api', version: '1.8.0-beta4') {
```

Comment:
I like this description, it explains why this mysteriously turns up and causes havoc. Can we expose it in commons via api configuration?